locator/tablets.cc

```diff
@@ -728,6 +728,16 @@ load_stats& load_stats::operator+=(const load_stats& s) {
     for (auto& [host, cap] : s.capacity) {
         capacity[host] = cap;
     }
+    if (s.critical_disk_utilization) {
+        if (!critical_disk_utilization) {
+            critical_disk_utilization = s.critical_disk_utilization;
+        } else {
+            for (auto& [host, cdu] : s.critical_disk_utilization.value()) {
+                critical_disk_utilization.value()[host] = cdu;
```

Comment:
How about decommissioned/removed/replaced nodes?
Don't we need to delete their data?
Is it reset automatically on other events?

Reply:
That is a good question. As I see, the `loads_stats` are created from scratch in `service::storage_service::load_stats_for_tablet_based_tables`. So eventually, the data for these nodes will not be present.

Even if we do have this outdated data for a while, we do not migrate tablets to the nodes you mentioned, don't we? @tgrabiec, please correct me if I am wrong here.